app/uk/gov/hmrc/essttpstubs/repo/EligibilityEntry.scala

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.essttpstubs.repo
+
+import essttp.journey.model.ttp.EligibilityCheckResult
+import julienrf.json.derived
+import play.api.libs.json.{Json, OFormat, OWrites}
+
+import java.time.Instant
+
+final case class EligibilityEntry(eligibilityCheckResult: EligibilityCheckResult) {
```

Comment:
it'd make more sense to have `createdAt` as a val in the parameter list of this case class I think. Otherwise, if you read this from mongo then suddenly the value of `createdAt` will be different from the one that was actually stored and if you store it again then you'd be storing the new value instead of the original one